packages/web-components/src/menu/menu.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { css } from '@microsoft/fast-element';
+import {} from '../theme/design-tokens.js';
+
+/** Menu styles
+ * @public
+ */
+export const styles = css`
+  :host {
+    position: relative;
+  }
+  .positioning-container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    transform: translate(0, 0);
```

Comment:
Is there any benefit to adding a z-index style here to ensure this appears above any other elements on the page? We could set it to a CSS variable that a user could override if necessary. @miroslavstastny and @chrisdholt does this make sense, or am I overthinking it?

Reply:
This definitely makes sense to me. The storybook story already shows the problem:
![image](https://github.com/microsoft/fluentui/assets/9615899/34615ad8-ca26-476a-8238-1749dc93b1fa)

I would start with something like
```css
z-index: var(--menu-z-index, 1)
```